superset/reports/schemas.py

```diff
@@ -320,7 +320,7 @@ class ReportSchedulePutSchema(Schema):
     log_retention = fields.Integer(
         metadata={"description": log_retention_description, "example": 90},
         required=False,
-        validate=[Range(min=1, error=_("Value must be greater than 0"))],
+        validate=[Range(min=0, error=_("Value must be 0 or greater"))],
```

Comment:
What does a log retention of `0` mean and how does this differ from `None`?

Reply:
@john-bodley I had forgotten to push some changes up. This should be covered in the latest push.